### ISSUE TITLE
PartDesign: Subsequent attempts to create fillets are created incorrectly or get aborted

### ISSUE THREAD

**Issue by CJVienna** [FreeCAD/FreeCAD#17452]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Problem description

Misbehaviour occurs even with simple objects (probably the same cause, but different manifestations).
Samples:
1. Start with a padded rectangle ... select any edge ... make a fillet ... ok ... select an adjacent edge ... make a fillet ... oops, all three edges (those with the same vertex in common) become rounded. Unfortunately I can't remove the unwanted one later on.
2. Start with a padded rectangle ... select any face ... make a fillet ... ok ... select any adjacent edge ... make a fillet ... oops, "Fillet001: BRep_API: command not done" --> "Error!" "Body: Tip shape is empty"

Unsatisfying workaround: Select all adjacent faces / edges where you want fillets applied and make fillet ... ok, but with the disadvantage you can't apply corrections later on without sometimes breaking referenced objects

Chamfer as a similar tool does not show those strange behaviours

### Full version info

```shell
OS: Windows 10 build 19045
Word size of FreeCAD: 64-bit
Version: 1.1.0dev.39024 (Git)
Build type: Release
Branch: main
Hash: cf8ad6637351819dfa07d49cfd65d511fe14a084
Python 3.11.9, Qt 5.15.13, Coin 4.0.3, Vtk 9.2.6, OCC 7.7.2
Locale: German/Austria (de_AT)
Stylesheet/Theme/QtStyle: unset/unset/Qt default
Installed mods: 
  * ArchTextures
  * Assembly4 0.11.12

same behaviour as in previous version 0.21.2:
[code]
OS: Windows 10 build 19045
Word size of FreeCAD: 64-bit
Version: 0.21.2.33771 (Git)
Build type: Release
Branch: releases/FreeCAD-0-21
Hash: b9bfa5c5507506e4515816414cd27f4851d00489
Python 3.8.10, Qt 5.15.2, Coin 4.0.1, Vtk 8.2.0, OCC 7.6.3
Locale: German/Austria (de_AT)
Installed mods: 
  * ArchTextures
  * Assembly4 0.11.12
[/code]
```

### Subproject(s) affected?

PartDesign

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by luzpaz** [comment]
@CJVienna did you cross-ref preexisting fillet bugs before opening this one? https://github.com/FreeCAD/FreeCAD/issues?q=is%3Aopen+is%3Aissue+label%3A%22Topic%3A+Fillets%22

**Comment by CJVienna** [comment]
@luzpaz yes, of course, I checked the preexisting fillet bugs. (and now I did again).

Really? An example file request for a simple pad?
Ok, there are two sample files attached in a ZIP
[Errors.zip](https://github.com/user-attachments/files/17525642/Errors.zip):
**Fillet_1** with Body.Pad.Edge4 selected and make Fillet executed. If you try to make an additional fillet on Body.Fillet.Edge4 then Body.Fillet.Edge13 unintendedly gets also a fillet.
**Fillet_2** with Body.Pad.Face6 selected and make Fillet executed. If you try to make an additional fillet on Body.Fillet.Edge4 then the error described in the problem description occurs.

B.t.w: I tried to also create a macro, but macro execution fails in V1.0 with <class 'NameError'>: name 'ActiveSketch' is not defined. But that is a different story.

**Comment by luzpaz** [comment]
> B.t.w: I tried to also create a macro, but macro execution fails in V1.0 with <class 'NameError'>: name 'ActiveSketch' is not defined. But that is a different story.

Thanks, do you mind opening ticket for that? TIA

**Comment by CJVienna** [comment]
> > B.t.w: I tried to also create a macro, but macro execution fails in V1.0 with <class 'NameError'>: name 'ActiveSketch' is not defined. But that is a different story.
> 
> Thanks, do you mind opening ticket for that? TIA

so I did (#17502)

**Comment by CJVienna** [comment]
Hi! The issue still exists.

Tested with latest weekly build.
OS: Windows 11 build 26100
Architecture: x86_64
Version: 1.1.0dev.14555 (Git shallow)
Build date: 2025/08/20 00:00:55
Build type: Release
Branch: main
Hash: 5e4fe3a3c3af46be530969d323368fee8bfcba3b
Python 3.11.13, Qt 6.8.3, Coin 4.0.3, Vtk 9.3.1, boost 1_86, Eigen3 3.4.0, PySide 6.8.3
shiboken 6.8.3, xerces-c 3.3.0, IfcOpenShell 0.8.2, OCC 7.8.1
Locale: German/Austria (de_AT)
Navigation Style/Orbit Style/Rotation Mode: Blender/Trackball/Drag at cursor
Stylesheet/Theme/QtStyle: FreeCAD Light.qss/FreeCAD Light/
Logical DPI/Physical DPI/Pixel Ratio: 96/93.2987/1

**Comment by FEA-eng** [comment]
Still relevant

```
OS: Windows 10 build 19045
Architecture: x86_64
Version: 1.1.0dev.14555 (Git shallow)
Build date: 2025/10/21 12:07:13
Build type: Release
Branch: main
Hash: 2096d0bd8603b429fdc0c746c34b3d73c3289fe1
Python 3.11.14, Qt 6.8.3, Coin 4.0.3, Vtk 9.3.1, boost 1_86, Eigen3 3.4.0, PySide 6.8.3
shiboken 6.8.3, xerces-c 3.3.0, IfcOpenShell 0.8.2, OCC 7.8.1
Locale: Polish/Poland (pl_PL)
Navigation Style/Orbit Style/Rotation Mode: CAD/Rounded Arcball/Window center
Stylesheet/Theme/QtStyle: unset/FreeCAD Light/fusion
Logical DPI/Physical DPI/Pixel Ratio: 96/90.0633/1
```

**Comment by FEA-eng** [comment]
Still relevant

```
OS: Windows 10 build 19045
Architecture: x86_64
Version: 1.2.0dev.20260505 (Git shallow)
Build date: 2026/05/05 22:34:03
Build type: Release
Branch: (HEAD detached at 08cacbbc2)
Hash: 08cacbbc27421e769be4ec423f9815a01e51e533
Python 3.11.15, Qt 6.8.3, Coin 4.0.8, Vtk 9.3.1, boost 1_86, Eigen3 3.4.0, PySide 6.8.3
shiboken 6.8.3, xerces-c 3.3.0, IfcOpenShell 0.8.2, OCC 7.8.1
Locale: Polish/Poland (pl_PL) [ OS: Polish/Poland (pl_PL) ]
Stylesheet/Theme/QtStyle: FreeCAD.qss/FreeCAD Light/
Navigation Style/Orbit Style/Rotation Mode: Blender/Rounded Arcball/Window center
Logical DPI/Physical DPI/Pixel Ratio: 96/90.0633/1
```

**Comment by CJVienna** [comment]
Still relevant.
Additional info: Fillet bug does not occur if selected edge is adjacent to a chamfer
Additional info: Apply a chamfer shows same type of bug if selected edge is adjacent to a fillet.

OS: Windows 11 build 26200
Architecture: x86_64
Version: 26.3.0dev.20260804 (Git shallow)
Build date: 2026/08/04 21:26:11
Build type: Release
Branch: (HEAD detached at 80923ab1b)
Hash: 80923ab1be1f31fff0a54db956177d582a658bab
Python 3.11.14, Qt 6.8.3, Coin 4.0.10 (bundled), Pivy 0.6.11 (bundled), Vtk 9.3.1, boost 1_86, Eigen3 3.4.0, PySide 6.8.3
shiboken 6.8.3, xerces-c 3.3.0, Clipper2 , IfcOpenShell 0.8.2, OCC 7.8.1
Locale: German/Austria (de_AT) [ OS: German/Austria (de_AT) ]
Stylesheet/Theme/QtStyle: FreeCAD.qss/FreeCAD Light/
QPA/File dialog/Color dialog: windows/via Win32/via Qt
Navigation Style/Orbit Style/Rotation Mode: Blender/Trackball/Drag at cursor
Logical DPI/Physical DPI/Pixel Ratio: 96/93.2987/1
OpenGL version/vendor/renderer: 4.6.0 NVIDIA 591.86/NVIDIA Corporation/NVIDIA GeForce RTX 5070 Ti/PCIe/SSE2
Installed mods:
  * Fasteners Workbench 0.5.41

**Comment by luzpaz** [comment]
@pieterhijma is this OCC related ?
